pkg/controller/v1alpha2/experiment/util/api_util.go

```diff
@@ -46,7 +48,18 @@ func DeleteExperimentInDB(instance *experimentsv1alpha2.Experiment) error {
 }
 
 func UpdateExperimentStatusInDB(instance *experimentsv1alpha2.Experiment) error {
-
+	newStatus := &api_pb.ExperimentStatus{
+		StartTime:      convertTime2RFC3339(instance.Status.StartTime),
+		CompletionTime: convertTime2RFC3339(instance.Status.CompletionTime),
+		Condition:      getCondition(instance),
+	}
+	request := &api_pb.UpdateExperimentStatusRequest{
+		NewStatus:       newStatus,
+		ExperimentName: instance.Name,
```

Comment:
format here is not gofmted, can you run gofmt for the file?

Reply:
done! thanks